.changeset/late-shoes-try.md

```diff
@@ -0,0 +1,6 @@
+---
+'@microsoft/atlas-site': major
+'@microsoft/atlas-css': major
```

Comment:
Just a reminder to update these values to `minor` after rework

Reply:
Done!